web/src/router.ts

```diff
@@ -112,6 +112,20 @@ const routes: RouteRecordRaw[] = [
       {
         path: 'build/:pipelineId',
```

Comment:
Did you tried `build/:rest(.*)`?

Reply:
Couldn't we use a function with doing `to.path.replace(/^build\//, 'pipeline/');`?